app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (VaccineError, NotWearingMaskError)
```

Comment:
Consider importing `OutdatedVaccineError` as well, since it is a specific type of `VaccineError` that might be useful for more granular exception handling if needed in the future.